### DISCUSSION TITLE
[Feature request] Add/assign apps to multiple users at the same time

### DISCUSSION THREAD

**Discussion by 2600box** [linuxserver/Heimdall#982] [Ideas]
Hello,

I googled for this and ended up editing the sqlite directly, but I may have missed something simple.

It would be great to be able to assign apps to multiple users when adding, or when editing.

The use case is say adding a recipe app to multiple users, at the moment it is necessary to go and add it to each user individually.

Would also be useful to have shared tags.

Thanks for making Heimdall, it is great!

**Comment by stillie** [comment]
I would love this aswell... am currently looking for a tool that does this. I use Heimdall at home but i have a couple links i would share with family members without having to add them again on their accounts

**Comment by HarHarLinks** [comment]
Similarly, I would like to have a public Heimdall dashboard for my friends and a private one with admin items for myself, where the private one should also inherit the public one's items.

**Comment by gabrieldevsouza** [comment]
Any plans for this? I would love to have this too!

**Comment by Midnex** [comment]
Just checking in to see if this is something that may be added later. Would love this as I am wanting to use it for multiple people within my house and possible a replacement for myapps in a small business.
